EpicLoot/src/Magic/MagicItemEffects/ModifyDrawStamina.cs

```diff
@@ -0,0 +1,22 @@
+﻿using HarmonyLib;
+
+namespace EpicLoot.MagicItemEffects
+{
+    // public float GetMaxDurability(int quality) =>
+    //   this.m_shared.m_maxDurability + (float) Mathf.Max(0, quality - 1) * this.m_shared.m_durabilityPerLevel;
+    [HarmonyPatch(typeof(ItemDrop.ItemData), nameof(ItemDrop.ItemData.GetDrawStaminaDrain))]
+    public static class ModifyDrawStamina_ItemData_GetDrawStaminaDrain_Patch
+    {
+        public static void Postfix(ItemDrop.ItemData __instance, ref float __result)
```

Comment:
Refactor this similar to the recommendation for GetAttackHealth. Change the m_drawStaminaDrain in a prefix, let run, then change back to the original similar to UpdateDodge.

Reply:
Finished in [commit](https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/5b885e9615288a1117bd17569c590ca2bd66dd9e)